src/coreclr/jit/unwind.h

```diff
@@ -131,6 +140,8 @@ class UnwindCodesBase
         return b >= 0xFD;
 #elif defined(TARGET_ARM64)
         return (b == UWC_END); // TODO-ARM64-Bug?: what about the "end_c" code?
+#elif defined(TARGET_LOONGARCH64)
```

Comment:
identical to arm64 case; add ifdef there

Reply:
OK, Thanks.